### DISCUSSION TITLE
[Prettier Plugin] Is it possible to get Prettier to only sort the classes?

### DISCUSSION THREAD

**Discussion by airblade** [tailwindlabs/tailwindcss#10342] [Help]
I'd like all my classes to be sorted into the correct order.  It looks like [the way to do that is with Prettier](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier#how-classes-are-sorted).

I don't use Prettier normally and don't want it to change anything other than the class order.  Is that possible?

(I tried to turn off all its built-in formatting so it only used the Tailwind plugin, but wasn't able to.)

Many thanks!

**Comment by airblade** [comment]
I presume this isn't possible but I thought I'd ask just in case.

**Comment by yanggggjie** [comment]
Hi, I encounter the same issue. And I  use a standalone edior  extension to sort class, for vscode  use [  headwind  ](https://marketplace.visualstudio.com/items?itemName=heybourn.headwind) and webstorm  use [tailwind-formatter](https://plugins.jetbrains.com/plugin/13376-tailwind-formatter)

**Comment by airblade** [comment]
I wrote a script to do this without Prettier: https://github.com/airblade/tcs.

**Comment by svoop** [comment]
There's also [Rustywind](https://github.com/avencera/rustywind).
